repos/fdbt-site/src/pages/createCaps.tsx

```diff
@@ -38,64 +59,259 @@ const CreateCaps = ({
+                                                        errorClass="govuk-input--error"
+                                                        hideText
+                                                        addFormGroupError={false}
+                                                    >
+                                                        <input
+                                                            className="govuk-input govuk-input--width-4 govuk-currency-input__inner__input"
+                                                            name="capPrice"
+                                                            data-non-numeric
+                                                            type="text"
+                                                            aria-describedby="cap-price-hint"
+                                                            id="cap-price"
+                                                            defaultValue={
+                                                                userInput && userInput.cap.price
+                                                                    ? userInput.cap.price
+                                                                    : ''
+                                                            }
+                                                        />
+                                                    </FormElementWrapper>
+                                                </div>
+                                            </div>
+                                        </>
+                                    </FormGroupWrapper>
+                                </div>
+                                <div className="govuk-!-margin-left-2 govuk-!-margin-right-2">
+                                    <FormGroupWrapper
+                                        errors={errors}
+                                        errorIds={['cap-period-duration-quantity', 'cap-period-duration-unit']}
+                                        hideErrorBar
+                                    >
+                                        <>
+                                            <label className="govuk-label" htmlFor="cap-period-duration">
+                                                Cap duration
+                                            </label>
+                                            <span className="govuk-hint" id="cap-duration-hint">
+                                                For example, 3 days
+                                            </span>
+                                            {/* */}
+
+                                            <div className="govuk-input__wrapper expiry-selector-wrapper">
+                                                <>
+                                                    <FormElementWrapper
+                                                        errors={errors || []}
```

Comment:
do we need to default this to an empty array? isnt that done in the server side props?